src/model/FileOperations.cpp

```diff
@@ -137,16 +137,17 @@ namespace model {
       }
     }
 
-    for (const auto& dirEnt : openstudio::filesystem::recursive_directory_iterator{src}) {
-      const auto& path = dirEnt.path();
-      auto relativePathStr = path.string();
-      boost::replace_first(relativePathStr, src.string(), "");
+    // Not using PathHelper's copyDirectory because we want clear log messages
+    for (const auto& dirEnt : openstudio::filesystem::directory_iterator{src}) {
+      const auto& srcFolderPath = dirEnt.path();
+      const auto& relativeFolderPath = openstudio::filesystem::relative(srcFolderPath, src);
 
       try {
-        openstudio::filesystem::copy(path, dest / relativePathStr);
+        // No need to pass the copy_options::overwrite_existing here, we wiped the directory before
+        openstudio::filesystem::copy(srcFolderPath, dest / relativeFolderPath, openstudio::filesystem::copy_options::recursive);
       } catch (const std::exception& e) {
         LOG_FREE(Error, "replaceDir",
-                 "Error copying from: " << toString(path) << " to: " << toString(dest / relativePathStr) << " Description: " << e.what());
+                 "Error copying from: " << toString(srcFolderPath) << " to: " << toString(dest / relativeFolderPath) << " Description: " << e.what());
```

Comment:
Fixup issue I found, it would throw that it couldn't copy the file because it existed. I think that's mostly because of the change to boost 1.79